R/use_tic.R

```diff
@@ -41,8 +61,10 @@ use_tic <- function(path = ".", quiet = FALSE) {
 
     #' 1. Enable deployment (if necessary, depending on repo type)
     #'    via [travis::use_travis_deploy()]
+    cli::cat_boxx(c("Step #4: We create a SSH key pair", "to allow Travis deployment to Github."), col = "green")
     if (needs_deploy(repo_type)) travis::use_travis_deploy()
 
+    cli::cat_boxx(c("Step #5: We create a Github PAT key on Travis CI to avoid", "Github API rate limitations in the builds."), col = "green")
```

Comment:
```suggestion
    cli::cat_boxx(c("Step #5: We create a Github PAT key on Travis CI", "to avoid Github API rate limitations in the builds."), col = "green")
```